elements/lisk-chain/src/chain.ts

```diff
@@ -12,7 +12,7 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 
-import { codec, Schema } from '@liskhq/lisk-codec';
+import { codec, MinimalSchema } from '@liskhq/lisk-codec';
```

Comment:
Maybe `GenericSchema` or `BasicSchema` or `RootSchema` as I think this always represents a full schema? No strong opinion and can be left as is.